app/frontend/components/domains/requirement-template/screens/jurisdiction-edit-digital-permit-screen/index.tsx

```diff
@@ -112,117 +111,109 @@ export const JurisdictionEditDigitalPermitScreen = observer(function Jurisdictio
-                        requirementBlockId
-                      ]
-                    }
-                  />
-                )
-              }}
-              requirementBlockCustomizations={watchedCustomizations?.requirementBlockChanges}
-              hideElectiveField={(requirementBlockId, requirement) => {
-                const customization = watchedCustomizations.requirementBlockChanges[requirementBlockId]
-
-                if (!customization || !customization.enabledElectiveFieldIds) {
-                  return true
-                }
-
-                return !customization?.enabledElectiveFieldIds?.includes(requirement.id)
-              }}
-            />
+            <ButtonGroup>
+              <Button
+                variant={"primary"}
+                rightIcon={<CaretRight />}
+                onClick={onSubmit}
+                isDisabled={isSubmitting || !isValid}
+                isLoading={isSubmitting}
+              >
+                {t("ui.publish")}
+              </Button>
+              <Button variant={"secondary"} onClick={onClose} isDisabled={isSubmitting}>
+                {t("ui.close")}
+              </Button>
+            </ButtonGroup>
           </Flex>
+          <BuilderTopFloatingButtons />
+          <SectionsDisplay
+            sections={templateSections}
+            isCollapsedAll={isCollapsedAll}
+            setSectionRef={setSectionRef}
+            formScrollToId={formScrollToId}
+            renderEdit={({ denormalizedRequirementBlock }) => {
+              return (
+                <JurisdictionRequirementBlockEditSidebar
+                  requirementBlockCustomization={getRequirementBlockCustomization(denormalizedRequirementBlock.id)}
+                  requirementBlock={denormalizedRequirementBlock}
+                  onSave={onBlockEditSave}
+                  triggerButtonProps={{
+                    isDisabled: isSubmitting,
+                  }}
+                  onResetDefault={(requirementBlockId) =>
+                    jurisdictionTemplateVersionCustomization?.customizations?.requirementBlockChanges?.[
+                      requirementBlockId
+                    ]
+                  }
+                />
+              )
+            }}
+            requirementBlockCustomizations={watchedCustomizations?.requirementBlockChanges}
+            hideElectiveField={(requirementBlockId, requirement) => {
+              const customization = watchedCustomizations.requirementBlockChanges[requirementBlockId]
+
+              if (!customization || !customization.enabledElectiveFieldIds) {
+                return true
+              }
+
+              return !customization?.enabledElectiveFieldIds?.includes(requirement.id)
+            }}
+          />
         </Flex>
-        <BuilderBottomFloatingButtons onScrollToTop={scrollToTop} onCollapseAll={onCollapseAll} />
       </Flex>
-    </RemoveScroll>
+      <BuilderBottomFloatingButtons onToggleCallback={onToggleCollapseAll} />
```

Comment:
pass the state and toggle function directly from this parent component, instead of relying on callback to sync state. This sbould be done for all compoentns that use the floating buttons component